src/components/GlobalLearningResourcesPage/GlobalLearningResourcesFiltersFallback.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Skeleton, Stack, StackItem } from '@patternfly/react-core';
+import './GlobalLearningResourcesFilters.scss';
+
+const GlobalLearningResourcesFiltersFallback: React.FC = () => {
+  const skeletonWidths = [
+    '100%',
+    '50%',
+    '60%',
+    '80%',
+    '40%',
+    '90%',
+    '30%',
+    '85%',
+    '65%',
+    '55%',
+    '75%',
+    '45%',
+    '95%',
+    '35%',
+  ];
+
+  return (
+    <Stack
+      hasGutter
+      className="lr-c-global-learning-resources-page__filters pf-v5-u-p-lg"
+    >
+      {/* Dynamic StackItems */}
+      {skeletonWidths.map((width, index) => (
+        <StackItem
+          key={index}
+          className={`lr-c-global-learning-resources-page__filters--item-${
```

Comment:
https://www.npmjs.com/package/classnames

Reply:
There is no point in using classnames utility because there are not any conditions for applying the class names. The template literals do great job in constructing static classnames based on the index.

Reply:
I see you are right. I though it is conditional 🙂